specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -3637,181 +3966,496 @@
+          "description": "The local admin credentials."
+        },
+        "domainAdmin": {
+          "$ref": "#/definitions/KeyvaultCredentialProperties",
+          "description": "The domain admin credentials."
+        }
+      }
+    },
+    "KeyvaultCredentialProperties": {
+      "description": "Credentials kept in the keyvault.",
+      "type": "object",
+      "properties": {
+        "userName": {
+          "type": "string",
+          "description": "The user name to the account."
+        },
+        "passwordKeyVaultResourceId": {
+          "type": "string",
+          "description": "The keyvault resource id to the keyvault secrets."
+        },
+        "passwordSecretName": {
+          "type": "string",
+          "description": "The keyvault secret name the password is stored in."
+        }
+      }
+    },
+    "HostPoolUpdateParameters": {
+      "description": "Represents properties for a hostpool update.",
+      "type": "object",
+      "properties": {
+        "saveOriginalDisk": {
+          "type": "boolean",
+          "description": "Whether to save original disk. False by default."
+        },
+        "maxVMsRemovedDuringUpdate": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The maximum virtual machines to be removed during hostpool update."
+        },
+        "maintenanceAlerts": {
+          "type": "array",
+          "description": "The alerts given to customers for hostpool update.",
+          "items": {
+            "$ref": "#/definitions/MaintenanceAlertsProperties"
+          }
+        },
+        "logOffDelaySeconds": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Grace period before logging off users in seconds."
+        },
+        "logOffMessage": {
+          "type": "string",
+          "description": "Log off message sent to user for logoff."
+        },
+        "scheduledTime": {
+          "$ref": "#/definitions/ScheduledTimeProperties",
+          "description": "When set schedules the hostpool update at specific time."
+        }
+      }
+    },
+    "MaintenanceAlertsProperties": {
+      "type": "object",
+      "description": "Properties for host pool update maintenance alerts.",
+      "properties": {
+        "message": {
+          "type": "string",
+          "description": "The path to the legacy object to migrate."
+        },
+        "duration": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The duration of the hostpool update in seconds."
+        },
+        "beforeKickOff": {
+          "type": "boolean",
+          "description": "Whether to send maintenance alert after or before hostpool update. False by default."
+        }
+      }
+    },
+    "ScheduledTimeProperties": {
+      "type": "object",
+      "description": "When set schedules the hostpool update at specific time.",
+      "properties": {
+        "time": {
```

Comment:
in what format? UTC ? ISO-8601?

Reply:
should it be "datetime"?

Reply:
> in what format? UTC ? ISO-8601?

https://swagger.io/docs/specification/data-models/data-types/#string
date-time – the date-time notation as defined by RFC 3339, section 5.6, for example, 2017-07-21T17:32:28Z